src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -27,18 +27,32 @@
 from turicreate.toolkits._internal_utils import (_raise_error_if_not_sframe,
                                                  _numeric_param_check_range)
 
-def create(dataset, target, feature = None, model = 'resnet-50',
-           validation_set='auto', max_iterations = 10, verbose = True,
-           seed = None, batch_size=64):
+_DEFAULT_SOLVER_OPTIONS = {
+'convergence_threshold': 1e-2,
+'step_size': 1.0,
+'lbfgs_memory_level': 11,
+'max_iterations': 10}
+
+  def create(dataset, target, features=None, model = 'resnet-50',
+    l2_penalty=0.01, l1_penalty=0.0,
+    solver='auto', feature_rescaling=True,
+    convergence_threshold = _DEFAULT_SOLVER_OPTIONS['convergence_threshold'],
+    step_size = _DEFAULT_SOLVER_OPTIONS['step_size'],
+    lbfgs_memory_level = _DEFAULT_SOLVER_OPTIONS['lbfgs_memory_level'],
+    max_iterations = _DEFAULT_SOLVER_OPTIONS['max_iterations'],
+    class_weights = None,
+    validation_set = 'auto',
+    verbose=True,
+    seed=None,
+    batch_size=64):
+    
     """
     Create a :class:`ImageClassifier` model.
-
```

Comment:
Did you mean to delete all these blank lines? We should leave them.

Reply:
I see what caused this. I copied this from the image_classifier.py file from the master branch, but when pasting code, GitHub seems to automatically remove blank lines.